src/status_im2/contexts/onboarding/events.cljs

```diff
@@ -140,12 +140,10 @@
      {:title               (i18n/label :t/multiaccount-exists-title)
       :content             (i18n/label :t/multiaccount-exists-content)
       :confirm-button-text (i18n/label :t/unlock)
-      :on-accept           #(do
-                              (re-frame/dispatch [:pop-to-root :profiles])
-                              ;; FIXME(rasom): obviously not cool
-                              (reset! profiles.view/show-profiles? false)
```

Comment:
While refactoring the code I noticed this FIXME was due to the show-profiles global atom.
I refactored the code so the global atom is no longer used, I'll refer to the global atom in a further comment.